docs/developer/base/logs-crawlers.md

```diff
@@ -0,0 +1,46 @@
+# Log Crawlers
+
+## Overview
+
+Some systems expose their logs from HTTP endpoints instead of files that the Logs Agent can tail.
+In such cases, you can create an agent integration to crawl the endpoints and submit the logs.
+
+This diagram shows how a logs crawler agent integration that fits into the agent.
```

Comment:
```suggestion
The following diagram illustrates the integration of a logs crawler agent within the Datadog Agent.
```
I think this is the best phrasing, if i'm understanding the concept correctly :)

Reply:
@aliciascott Hmm, "logs crawler agent" isn't a thing.

Tell me what you think of this phrasing:

> The following diagram illustrates how crawling logs integrations into the Datadog Agent.

Reply:
Ok yes I think that is fine, but I think it would be "integrates" used in that sentence ?

Reply:
yes, I got lost between all the "integration" and "integrates" variants :joy